build.gradle

```
@@ -57,7 +57,7 @@ configurations {
 }
 
 dependencies {
-    ktlint ("com.pinterest:ktlint:0.48.2") {
+    ktlint ("com.pinterest:ktlint:0.49.0") {
```

Pinning this for now - will update in a follow up PR.

```suggestion
    ktlint ("com.pinterest:ktlint:0.48.2") {
```